trainingsanleitungen/scratch/scratch-fang-mich-v3.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: sushi
+title: Scratch Fang-Mich
+description: In diesem Spiel musst du versuchen, dem Haifisch zu entkommen.
+---
+
+# Scratch Fang-Mich
+
+In diesem Spiel bist du ein Fisch, der dem großen Haifisch entkommen muss. Schaffst du es?
+
+## Bühne und Figuren anlegen
+
+1. ![Bühnenbild auswählen](scratch-fang-mich-v3/01-background.png){: .right}
+Als erstes legst du fest, wie dein Spielfeld aussehen soll. Wir brauchen zuerst das Aquarium, in dem die Fische schwimmen. Wähle als erstes rechts unten unter *Bühnenbild wählen* ein Bühnenbild aus, zum Beispiel ein Underwater 1.
+
+2. ![Figur löschen](scratch-fang-mich-v3/02-delete-scratchy.png){: .right}
+Als nächstes lösche die Figur Scratchy mit dem Namen *Sprite 1* indem du mit der rechten Maustaste darauf klickst. 
+Im angezeigten Menü kannst du Scratchy löschen.
+
+3. ![Figuren hinzufügen](scratch-fang-mich-v3/03-fish.png){: .right}
+Jetzt brauchen wir einen Haifisch sowie einen Fisch, mit dem wir dem Haifisch entkommen wollen. Klicke dazu auf *Figur wählen* und füge einen Fisch sowie einen Haifisch dazu. Natürlich können es auch andere Figuren sein, zum Beispiel ein Käfer, der dem Vogel davonläuft oder eine Person die dem Löwen davon läuft.
```

Comment:
* Zu *Käfer* und *Vogel* würde *davonfliegt* besser passen. 
* *davon läuft* schreibt man [zusammen](https://www.duden.de/rechtschreibung/davonlaufen), also *davonläuft*.

Beides war schon in der ursprünglichen Vorlage falsch.